src/Lykke.SettingsReader/SettingsTemplate/IJsonSettingsTemplateGenerator.cs

```diff
@@ -0,0 +1,13 @@
+namespace Lykke.SettingsReader.SettingsTemplate;
+
+/// <summary>
+/// Interface for IJsonSettingsTemplateGenerator
+/// </summary>
+public interface IJsonSettingsTemplateGenerator
+{
+    /// <summary>
+    /// This method will generate JsonSettingsTemplate in json. Default implementation is taking IConfiguration interface and converts it into json settings template.
```

Comment:
There is no default implementation in interface so the comment about default implementation is probably better goes with `JsonSettingsTemplateGenerator` class.
```suggestion
    /// This method will generate JsonSettingsTemplate in json. 
```